TradeItIosTicketSDK2/TradeItPortfolioEquityPositionsTableViewCell.swift

```diff
@@ -97,6 +97,12 @@ class TradeItPortfolioEquityPositionsTableViewCell: UITableViewCell {
     // MARK: private
 
     private func updateTradeButtonVisibility() {
+        guard let _ = self.selectedPosition?.linkedBrokerAccount.orderCapabilities(forInstrument: TradeItTradeInstrumentType.equities) else {
```

Comment:
There is a `tradeable` flag in the account response we don't have in the `TradeItLinkedBrokerAccount`. Should we add and use this flag instead of checking the capabilities?